motion-1/Overview.bs

```diff
@@ -92,30 +92,41 @@ In addition to the property-specific values listed in their definitions, all pro
 
 ISSUE: Add more details and examples.
 
-<h3 id="motion-path-property">Motion path: The 'motion-path' property</h3>
+<h3 id="offset-path-property">Define a path: The 'offset-path' property</h3>
 
 <pre class='propdef'>
-Name: motion-path
-Value: <<url>> | [ <<basic-shape>> | <<path()>> ] || <<geometry-box>> | none
-Initial: none
+Name: offset-path
+Value: <<url>> | <<angle>> && contain? | [ <<basic-shape>> | <<path()>> ] || <<geometry-box>> | none
+Initial: 0deg for <<angle>>, otherwise none
```

Comment:
The initial value can't be dependent upon type like this - just make it always 'none' and treat none as 0deg if required.